src/extensions/default/JavaScriptCodeHints/node/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "brackets-javascript-code-hints",
+  "dependencies": {
+    "acorn": "3.3.0",
+    "tern": "0.20.0"
+  },
+  "scripts": {
+    "postinstall": "node ./fix-acorn"
```

Comment:
@swmitra 
this script fails when trying to run `npm install`
I don't think it's needed if tern doesn't run in webworker anymore